### DISCUSSION TITLE
Bring #4644 into the next release for SSO and RBAC using Google

### DISCUSSION THREAD

**Discussion by covidium** [argoproj/argo-workflows#4929] [Ideas]
In the latest release (v2.12.5) we have the possibility to enable SSO + RBAC, but the rbac-rule annotation for filtering on email addresses doesn't work. I've tried to run the deployment with the latest tag and that works. Therefore my question is: when can the MR #4644 be included in a release? Are there issues around it that I've missed?

Best regards,
Ovidiu

**Comment by alexec** [comment]
This should be an issue, not a discussion I think.

**Comment by covidium** [comment]
Closed discussion and issue reported under number: #4942
